src/Controllers/ApiController.php

```diff
@@ -3,6 +3,18 @@
 namespace Engelsystem\Controllers;
 
 use Engelsystem\Http\Response;
+use Engelsystem\Http\Exceptions\HttpForbidden;
+use Symfony\Component\HttpFoundation\JsonResponse;
+use Symfony\Component\HttpFoundation\Request;
+
+# Legacy functions
+/* Doing their job, but adding a proper namespaces breaks legacy code :( */
```

Comment:
That's the point where this code can't be fully unit tested as it depends on legacy code that has no code coverage and depends on database entries that can't be used in a in memory sqlite database. The only thing where i could see them be used would be some kind of factory class but that feels more like a hack than a good workaround as  is the main point why no API exists at the moment

Reply:
That's correct. We won't get test coverage for this, but the controller has no side-effects to anything else. 
Should it break the api isn't working, which is always the case if the api isn't implemented at all.
The controller could be moved to /includes as well if 100% coverage for /src is more important than the project structure.